### PR TITLE
[tools] Remove unsupported rootfs install ubuntu version

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit removes install_rootfs.sh arguments trusty (ubuntu 14.04) and related files

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: #5762

### CODE REVIEW

**Review by seanshpark — DISMISSED** [review]
LGTM

File: tools/cross/install_rootfs.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ usage()
 {
     echo "Usage: $0 [BuildArch] [LinuxCodeName] [--setproxy=IP] [--skipunmount]"
     echo "BuildArch can be: arm(default), aarch64 and armel"
-    echo "LinuxCodeName - optional, Code name for Linux, can be: bionic(default), trusty, xenial, focal"
+    echo "LinuxCodeName - optional, Code name for Linux, can be: bionic(default), xenial, focal"
```
Comment (lemmaa) [inline tools/cross/install_rootfs.sh:8]: It's trivial, but how about changing the order to xenial, bionic(default), focal?
Anyway, LGTM regardless of it.

**Review by lemmaa — DISMISSED** [review]

File: tools/cross/install_rootfs.sh (hunk 1 above)
Comment (hseok-oh) [reply tools/cross/install_rootfs.sh:8]: OK. I'll update it.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by lemmaa — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**